plugins/occt/module/vtkF3DOCCTReader.cxx

```diff
@@ -143,7 +143,8 @@ class vtkF3DOCCTReader::vtkInternals
         {
           std::array<unsigned char, 3> rgb = { 0, 0, 0 };
           Quantity_Color aColor;
-          if (this->ColorTool->GetColor(edge, XCAFDoc_ColorCurv, aColor))
+          if (this->ColorTool->GetColor(edge, XCAFDoc_ColorCurv, aColor) ||
```

Comment:
Oh ok, so the logic is to fallback on the shape color if there's no color on the cell?

Reply:
You described it in the PR description, I got it 👍

Reply:
hopefully we don't have to fallback higher up on compound shapes and whatnot... I guess we'll see if examples show up in the wild